### PR TITLE
[Security Rules] Update security rules package to v0.13.3

### PR DESCRIPTION
**Pull request by rw-access**
## What does this PR do?
Update the Security Rules package to version 0.13.3.
Autogenerated from commit  https://github.com/elastic/detection-rules/tree/0

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] ~I have verified that all data streams collect metrics or logs.~
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist
- Install the most recently release security rules in the Detection Engine
- Install the package
- Confirm the update is available in Kibana. Click "Update X rules" or "Install X rules"
- Look at the changes made after the install and confirm they are consistent

## How to test this PR locally
- Perform the above checklist, and use `package-storage` to build EPR from source

## Related issues
None

## Screenshots
None

### CODE REVIEW

File: packages/security_detection_engine/changelog.yml [hunk 1]
```diff
@@ -1,50 +1,55 @@
 # newer versions go on top
-# NOTE: please use prerelease versions until a package is ready for production
```
Comment (brokensound77) [inline packages/security_detection_engine/changelog.yml:2]: Are these changelog updates going to normalize from here on out with the rules workflow making the PR?

**Review by brokensound77 — APPROVED** [review]
LGTM 👍

File: packages/security_detection_engine/changelog.yml (hunk 1 above)
Comment (rw-access) [reply packages/security_detection_engine/changelog.yml:2]: yeah I think the diffs will be small for this file going forward

**Comment by brokensound77** [comment]
>Autogenerated from commit https://github.com/elastic/detection-rules/tree/0

I know the bug was fixed, but do you remember the commit it pulled from to update the description

**Pull request merged**